app/src/main/java/io/left/meshim/models/Message.java

```diff
@@ -153,10 +164,13 @@ public Message(User sender, User recipient, String message, boolean isMyMessage)
      * @param isMyMessage if this device's user sent the message
      * @param date date message was sent
      */
-    public Message(int senderId, int recipientId, String message, boolean isMyMessage, Date date) {
+    public Message(int senderId, int recipientId, String message, boolean isMyMessage, Date date,String filePath,String fileName) {
```

Comment:
It's better to have your constructor variables always have the same order. E.g. `Foo(A); Foo(A, B); Foo(A, B, C, D);`. Here you have `Foo(A), Foo(A, C, D); Foo(A, B, C, D)`, where we have to remember the constructor that has `Date` puts it in the middle instead of on the end.